src/main/kotlin/org/jitsi/rtp/RtpTwoByteHeaderExtension.kt

```diff
@@ -40,7 +40,15 @@ class RtpTwoByteHeaderExtension(val buf: ByteBuffer) : RtpHeaderExtension() {
 
     companion object {
         const val HEADER_SIZE = 2
-        const val COOKIE: Short = 0x1000.toShort()
+        const val COOKIE: Short = 0x1000
+
+        /**
+         *  Checks if `headerExtensionType` matches the IDs used by
+         *  [RtpTwoByteHeaderExtension]. See
+         * https://tools.ietf.org/html/rfc5285#section-4.3
+         */
+        fun headerExtensionTypeMatches(headerExtensionType: Short): Boolean
+                = COOKIE.compareTo(headerExtensionType.toInt() and 0xfff0) == 0
```

Comment:
why mask with `0xfff0`?  maybe i'm missing something in the spec, but the header extension type cookie is a full 2 bytes from what i see?

Reply:
Ah, now I see.